### PR TITLE
Story/csi 1135 vol cap validation fix

### PR DESCRIPTION
**Pull request by igorgonibm**
Fix volume block/fs validation in con troller

### CODE REVIEW

File: controller/tests/controller_server/utils_test.py [hunk 1]
```diff
@@ -61,12 +62,6 @@ def test_validate_volume_capabilities(self):
         with self.assertRaises(ValidationException):
             utils.validate_csi_volume_capabilties([caps])
 
-        caps = Mock()
-        caps.mount = None
-        caps.access_mode.mode = access_types.SINGLE_NODE_READER_ONLY
-        with self.assertRaises(ValidationException):
```
Comment (oriyarde) [inline controller/tests/controller_server/utils_test.py]: why is this removed?

Comment (igorgonibm) [reply controller/tests/controller_server/utils_test.py]: mount can never be none
The parallel test is added to check block. Which makes HasField return false instead of setting mount to None

File: controller/controller_server/config.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 access_types = csi_pb2.VolumeCapability.AccessMode
```
[the remark this answers is not in the document]
Comment (igorgonibm) [reply controller/controller_server/config.py]: renamed

**Comment by oriyarde** [comment]
@igorgonibm can we delete this branch?
